nengo_loihi/hardware/builder.py

```diff
@@ -126,8 +126,6 @@ def build_core(n2core, core):  # noqa: C901
             if synapse.learning:
                 numStdp += len(axons)
                 assert np.all(axons >= firstLearningIndex)
-            # else:
-            #     assert np.all(axons < firstLearningIndex)
```

Comment:
I do want to remember to do this when we have better allocators. I'll find the allocator issue and add a note.